src/app/views/provisioning/summary.controller.coffee

```diff
@@ -8,20 +8,22 @@ angular.module 'mnoEnterpriseAngular'
     vm.selectedCurrency = MnoeProvisioning.getSelectedCurrency()
     vm.subType = if $stateParams.cart == 'true' then 'cart' else 'active'
     vm.subscription = MnoeProvisioning.getCachedSubscription()
-    vm.quoteBased = vm.subscription.product_pricing.quote_based
-    vm.quote = MnoeProvisioning.getCachedQuote() if vm.quoteBased
+
+    initSummary = () ->
+      vm.singleBilling = vm.subscription.product.single_billing_enabled
+      vm.billedLocally = vm.subscription.product.billed_locally
+      vm.quoteBased = vm.subscription.product_pricing?.quote_based
 
     if !_.isEmpty(vm.subscription)
+      initSummary()
+      vm.quote = MnoeProvisioning.getCachedQuote() if vm.quoteBased
       vm.isLoading = false
     else
       MnoeProvisioning.initSubscription({subscriptionId: $stateParams.subscriptionId})
         .then((response) ->
           vm.subscription = response
-          vm.singleBilling = vm.subscription.product.single_billing_enabled
-          vm.billedLocally = vm.subscription.product.billed_locally
-          vm.quoteBased = vm.subscription.product_pricing.quote_based
-          )
-        .finally(() -> vm.isLoading = false)
+          initSummary()
```

Comment:
Do we need to fetch the quote if it is not cached? e.g. if we refresh the page?

Reply:
I don't think we want to do that as you can hard code the url for the summary page. If the pricing changed since the last time they viewed the summary page we don't want them to see a different quote.